sourcegen-model/src/main/java/io/micronaut/sourcegen/model/ExpressionDef.java

```diff
@@ -337,6 +338,16 @@ default CallInstanceMethod invoke(MethodElement methodElement, List<ExpressionDe
         );
     }
 
+    /**
+     * The get property value expression.
+     *
+     * @param propertyElement The property element
+     * @return The get property value expression
+     * @since 1.3
+     */
+    default GetPropertyValue getPropertyValue(PropertyElement propertyElement) {
+        return new GetPropertyValue(this, propertyElement);
+    }
```

Comment:
It would be good if `VariableDef.Field` would resolve the same as this automatically (inside class it would be just property access, otherwise a getter). Having 2 choices seems like a complication.

If we have to keep this, I think `PropertyValue` or `PropertyValueGetter` would be more matching with the naming here.

Reply:
There is a TODO; we don't support an alternative to `invoke` for fields.
We should have `SetPropertyValue` in the future so that's why I kept the `Get`

Reply:
Makes sense